wandb/sdk/launch/builder/build.py

```diff
@@ -141,6 +137,12 @@ def get_docker_user(launch_project: LaunchProject, runner_type: str) -> Tuple[st
     {user} || echo ""
 """
 
+SAGEMAKER_ENTRYPOINT_TEMPLATE = """
+COPY ./src/train {workdir}
+RUN chmod +x {workdir}/train
+ENTRYPOINT ["sh", "train"]
```

Comment:
Hmm, a bit of a nit but there's no need to run `chmod +x` if we're running it explicitly with `sh`.

Reply:
nice catch